src/app/dim-ui/TileGrid.m.scss

```diff
@@ -15,22 +15,24 @@
 }
 
 .tile {
-  composes: flexColumn from '../dim-ui/common.m.scss';
-  border: 1px solid #666;
+  composes: flexRow from '../dim-ui/common.m.scss';
+  border: 1px solid transparent;
   cursor: pointer;
   padding: 8px;
-  display: flex;
-  flex-direction: row;
   gap: 8px;
   height: 100%;
   box-sizing: border-box;
+  position: relative;
   &:hover {
     background-color: rgba(255, 255, 255, 0.1);
   }
   &:focus-visible {
-    border-color: var(--theme-accent-primary);
+    border-color: var(--theme-accent-secondary);
     outline: none;
   }
+  > *:first-child {
+    float: left;
```

Comment:
I am always sus on floats, but I do know when used correctly they are fine. What is the purpose of it here, as it ins't clear in from the rest of the diff.

Reply:
Leftover from a different experiment